pytorch_lightning/trainer/trainer.py

```diff
@@ -255,8 +255,8 @@ def __init__(
 
             plugins: Plugins allow modification of core behavior like ddp and amp, and enable custom lightning plugins.
 
-            precision: Double precision (64), full precision (32) or half precision (16). Can be used on CPU, GPU or
-                TPUs.
+            precision: Double precision (64), full precision (32), half precision (16) or bfloat16 precision (bfloat16).
+                Can be used on CPU, GPU or TPUs.
```

Comment:
btw in our TPUHalfPrecisionPlugin we force bfloat if I understand correctly

Reply:
yep it's the only native precision for TPUs. Now using 16 or bfloat16 will do the same thing for TPUs